### DISCUSSION TITLE
10G PON (XGS-PON)

### DISCUSSION THREAD

**Discussion by jjeziorny** [Anime4000/RTL960x#192] [Ideas]
10G PON (XGS-PON) has starting to appear (at least here in Spain), so I was wondering if @Anime4000 has any plans to extend the amazing knowledge provided in this repository to such technology?

As a long-time user of GPON I can say this is probably the best place for anyone looking to replace provider equipment with their own GPON solution, so I think it would make sense to expand this repo into 10G PON technology.

**Comment by Anime4000** [comment]
I have started add some info about 10G PON, from XGS-PON and XG-PON
what I found from my friend, XG/XGS-PON ONU SFP module is quite expensive to buy, at least in my currency 😅
however, my friend said it use same command as Realtek xPON, it should be no problem to clone from old to new

but I haven't dive in deep yet, perhaps you buy the module and try, the feedback would be helpful

**Comment by jjeziorny** [comment]
I've contacted my ISP to see if they offer 10G in my area and unfortunately not yet,  but they plan to expand quickly and will let me know once available .

I'll research on SFPs in the meantime and if I find some not too expensive will buy to start playing 

Bought a TL-ER2260T last week (should arrive in couple weeks) as my existing Ubiquity ER-12 doesn't have SFP+ and want to see if I can bump my 1G connection to 2.5 and also be ready for 10G when available.
Bit worried as ,afaik, the web interface is Chinese only, but chrome translator should be enough I hope.

**Reply by Anime4000** [reply]
what I see Hisense 10G PON is in English, WebGUI is similar to ODI, should be identical when comes to command line options

**Comment by jjeziorny** [comment]
Cool, got one for 29USD to play around

**Reply by Anime4000** [reply]
Nice, try it and tell us how it goes!
